.eslintrc.js

```diff
@@ -31,17 +31,33 @@ module.exports = {
     'packages/core/config/storybook/**/*',
     'packages/studio/dist-*/**/*',
   ],
-  rules: {
-    '@typescript-eslint/no-explicit-any': 'off',
-    curly: 'error',
-    '@typescript-eslint/consistent-type-imports': 'error',
-  },
```

Comment:
I moved these into the `overrides` > `files: ['*.ts', '*.tsx']`. I figured it'd be simpler to reduce the number of `rules:` objects if possible.

Reply:
Sounds good to me!